MapboxCoreNavigation/RouteController.swift

```diff
@@ -262,6 +262,14 @@ open class RouteController: NSObject {
      */
     var rawLocation: CLLocation?
     
+    public var modifiedRerouteDistanceGivenIntersectionLocation: CLLocationDistance {
```

Comment:
Let’s call the concept of “how far can I go before triggering rerouting” a “rerouting tolerance”. So this property could be named simply `reroutingTolerance`. (The fact that it accounts for the distance from an intersection is an implementation detail.)